### PR TITLE
libXmu, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libxmu/libxmu-1.1.1.recipe [hunk 1]
```diff
@@ -0,0 +1,88 @@
+SUMMARY="X Session Management Library"
+DESCRIPTION="The Session Management Library (SMlib) is a low-level /"C/" \
```
Comment (fbrosson) [inline x11-libs/libxmu/libxmu-1.1.1.recipe]:
```diff
-DESCRIPTION="The Session Management Library (SMlib) is a low-level /"C/" \
+DESCRIPTION="The Session Management Library (SMlib) is a low-level \"C\" \
```

File: x11-libs/libxmu/libxmu-1.1.1.recipe [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	autoreconf -vfi
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -rf $libDir/*.la
```
Comment (fbrosson) [inline x11-libs/libxmu/libxmu-1.1.1.recipe]:
```diff
-	rm -rf $libDir/*.la
+	rm -f $libDir/*.la
```
(no need for recursion)

File: x11-libs/libxmu/libxmu-1.1.1.recipe [hunk 3]
```diff
@@ -0,0 +1,88 @@
+SUMMARY="X Session Management Library"
+DESCRIPTION="The Session Management Library (SMlib) is a low-level /"C/" \
+language interface to XSMP. It is expected that higher level toolkits, \
+such as Xt, will hide many of the details of session management from \
+clients. Higher level toolkits might also be developed for session managers \
+to use, but no such effort is currently under way."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
```
Comment (fbrosson) [inline x11-libs/libxmu/libxmu-1.1.1.recipe]:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
```

File: x11-libs/libxmu/libxmu-1.1.1.recipe (hunk 1 above)
Comment (Begasus) [reply x11-libs/libxmu/libxmu-1.1.1.recipe]: Jikes, forgot to update the DESCRIPTION

File: x11-libs/libxmu/libxmu-1.1.1.recipe (hunk 3 above)
Comment (Begasus) [reply x11-libs/libxmu/libxmu-1.1.1.recipe]: Will fix :)

File: x11-libs/libxmu/libxmu-1.1.1.recipe (hunk 2 above)
Comment (Begasus) [reply x11-libs/libxmu/libxmu-1.1.1.recipe]: missed this one also ...  thanks!

**Pull request merged**